frontend/src/locales/en.json

```diff
@@ -385,10 +386,11 @@
           "jubladb": "JublaDB",
           "midata": "MiData"
         },
-        "registernow": "Register now"
+        "registernow": "Register now",
+        "termsOfServiceLink": "terms of service"
       },
       "register": {
-        "acceptTermsOfUse": "Accept the terms of use",
+        "acceptTermsOfService": "Accept the terms of use",
```

Comment:
Hier ist noch der einzige Ort wo `terms of use` steht.
Sonst immer `terms of service`. Ist das Absicht?

Reply:
Nope, will change, thanks